jetty-server/src/test/java/org/eclipse/jetty/server/GracefulStopTest.java

```diff
@@ -631,7 +632,7 @@ public void handle(String target, Request baseRequest, HttpServletRequest reques
 
         // Check new connections rejected!
         String unavailable = connector.getResponse("GET / HTTP/1.1\r\nHost:localhost\r\n\r\n");
-        assertThat(unavailable, containsString(" 503 Service Unavailable"));
+        assertThat(unavailable, containsString(" 404 Not Found"));
```

Comment:
@gregw are you sure that this behaviour change is ok?

Reply:
The `StatisticsHandler` is not sending the 404, it is not handling the request and then the `HttpChannel` is sending a 404 because the request was not handled. Previously we were sending 503s from the `StatisticsHandler` now we are not.

If the `StatisticsHandler` is wrapping a `ContextHandler` then the `ContextHandler` won't be able to send any 503s as the request will never get there.

Reply:
Ah I wasn't looking closely enough.    But I still think we shouldn't change this behaviour, at least not in 9